src/main/java/com/autotune/analyzer/Analyzer.java

```diff
@@ -30,8 +31,10 @@ public static void start(ServletContextHandler contextHandler) {
 
         try {
             addServlets(contextHandler);
-            PerformanceProfilesDeployment.getPerformanceProfiles(); //  Performance profile should be called first
-            KruizeOperator.getKruizeObjects(kruizeOperator);
+            if (KruizeDeploymentInfo.local == true) {
+                PerformanceProfilesDeployment.getPerformanceProfiles(); //  Performance profile should be called first
+                KruizeOperator.getKruizeObjects(kruizeOperator);
+            }
```

Comment:
Since it adds a watcher, it will fail in the ROS environment and consequently fails to bring up the pod.
`
kubernetesServices.addWatcher(KubernetesContexts.getPerformanceProfileCrdContext(), performanceProfileObjectWatcher);`

Reply:
The watcher code was not added by the Kruize Local changes ... so not sure why this should change now?